frontend/src/components/CompanyList.jsx

```diff
@@ -54,8 +54,10 @@ const CompanyList = () => {
                 ticker: value.ticker,
                 daily: exploreCompaniesPrices[ind].price.toFixed(2),
                 dailyChange: (
-                  exploreCompaniesPrices[ind].price -
-                  exploreCompaniesPrices[ind].dayStart
+                  ((exploreCompaniesPrices[ind].price -
+                    exploreCompaniesPrices[ind].dayStart) /
+                    exploreCompaniesPrices[ind].dayStart) *
+                  100
                 ).toFixed(2),
```

Comment:
This operation was in another file -- possible candidate for utils?

Reply:
Also I can't comment on the above lines but didn't you replace the "-" with PLACEHOLDER?